ny-portal/src/app/(frontend)/layout.tsx

```diff
@@ -21,14 +19,13 @@ export default function PortalLayout({ children }: Props) {
                 >
                     Hopp til innhold
                 </Link>
-                <CookieConsentProvider statistics functional>
-                    <div className={styles.layout}>
-                        <Navigation />
+                <div className={styles.layout}>
+                    <Navigation />
+                    <CookieConsent>
```

Comment:
For noen som ikke var helt våken da denne filen landet her første gangen, hva oppnår vi med å wrappe hele applikasjonen i en `CookieConsent`?